src/lib/matrix/test/unwrap.cpp

```diff
@@ -0,0 +1,62 @@
+#include <matrix/math.hpp>
+#include <gtest/gtest.h>
+
+using namespace matrix;
+
+TEST(Unwrap, UnwrapFloats)
```

Comment:
I would do the naming like I suggested in https://github.com/PX4/PX4-Autopilot/pull/19250

Reply:
@MaEtUgR do you mean e.g. `UnwrapTest` ?

Reply:
Don't worry about the naming let's get this pr in first and I'll adopt when rebasing https://github.com/PX4/PX4-Autopilot/pull/19250